independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "Api Key",
+                    "description": "Api Key for this API.",
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#FDFDFD",
+        "capabilities": [
+            "actions"
+        ],
+        "publisher": "Sinergyne Global Communications, S.L.",
```

Comment:
@AfriSR Please provide your name as a publisher as you are publishing it as an independent publisher connector.